code/modules/mob/living/living.dm

```diff
@@ -305,23 +305,6 @@
 
 /mob/living/proc/revive()
 	rejuvenate()
-	if(iscarbon(src))
```

Comment:
Oh, I think I forgot to move this to a parent call? Why did I change this, it isn't related to the thrust of this PR.